src/AeroSandbox/src/interceptors/storage/sharedStorage.ts

```
@@ -1,4 +1,4 @@
-import config from "$aero/examples/config";
+var config: any = {}
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)